fs/dafsclient/rmtclient_impl.hpp

```diff
@@ -155,7 +155,7 @@ class CRemoteBase : public CSimpleInterfaceOf<IDaFsConnection>
     static unsigned         lastfailtime;
     DAFSConnectCfg          connectMethod;
 
-    void connectSocket(SocketEndpoint &ep, unsigned localConnectTime=0, unsigned localRetries=0);
+    void connectSocket(SocketEndpoint &ep, unsigned connectTimeoutMs=0, unsigned connectRetries=INFINITE);
```

Comment:
NB: changed default to max retries value, as 0 is a legitimate override value for retries.